docs/scripts/gen_beacon_rst.py

```diff
@@ -23,30 +25,118 @@
     canopen.objectdictionary.REAL32: "float32",
     canopen.objectdictionary.REAL64: "float64",
     canopen.objectdictionary.VISIBLE_STRING: "str",
-    canopen.objectdictionary.OCTET_STRING: "octect_str",
+    canopen.objectdictionary.OCTET_STRING: "octet_str",
     canopen.objectdictionary.DOMAIN: "domain",
 }
 """Nice names for CANopen data types."""
 
 
-def gen_beacon_rst(config: OreSatConfig, file_path: str):
+def gen_beacon_rst(config: OreSatConfig, file_path: str, url: str):
     """Genetate a rst file for a beacon definition."""
 
-    title = f"{ORESAT_NICE_NAMES[config.oresat_id]} Beacon Definition"
+    title = "Beacon Definition"
+    header_title = "AX.25 Header"
     lines = [
         f"{title}\n",
         f'{"=" * len(title)}\n',
         "\n",
-        ".. csv-table::\n",
-        '   :header: "Offset", "Card", "Data Name", "Data Type", "Size", "Description"\n',
+        f"YAML configuration file that defines this beacon can be found at: {url}\n",
+        "\n",
+        f"{header_title}\n",
+        f'{"-" * len(header_title)}\n',
         "\n",
     ]
 
+    c3_od = config.od_db[NodeId.C3]
+    src_callsign = c3_od["beacon"]["src_callsign"].value
+    src_callsign = src_callsign + " " * (6 - len(src_callsign))
+    src_ssid = c3_od["beacon"]["src_ssid"].value
+    dest_callsign = c3_od["beacon"]["dest_callsign"].value
+    dest_callsign = dest_callsign + " " * (6 - len(dest_callsign))
+    dest_ssid = c3_od["beacon"]["dest_ssid"].value
+    command = c3_od["beacon"]["command"].value
+    reversed_bits = 0b0110_0000
+    control = c3_od["beacon"]["control"].value
+    pid = c3_od["beacon"]["pid"].value
+
+    dest_ssid = (dest_ssid << 1) | (int(not command) << 7) | reversed_bits
+    src_ssid = (src_ssid << 1) | (int(command) << 7) | reversed_bits | 1
```

Comment:
Command packets should have only the dest C-Bit set, so I think the not from the line above needs to move down here.